### ISSUE TITLE
Question: Player.exe command line parameters

### ISSUE THREAD

**Issue by Alopexus** [framefield/tooll#23]
Is there comprehensive list of the existing command line parameters available that can be used when launching an exported project? I ask this as I am interested in creating a custom UI for launching a demo that replaces the standard T2 Player launch option window for Resolution, AspectRatio, etc. 

Thanks. Loving tooll2, fantastic software. :-)

**Comment by drcynic** [comment]
So far there are only two options: --hide_dialog and --time_logging take a look in https://github.com/framefield/tooll/blob/master/Player/CommandLineOptions.cs

All other options are configured with the startup dialog - so far. If you want to replace the dialog with an own you will need to fill ContextSettings (https://github.com/framefield/tooll/blob/master/Core/ContextSettings.cs). This is what https://github.com/framefield/tooll/blob/master/Player/StartUpDialog.cs does or you extend the command line options if you need to start it from another app/process.

**Comment by Alopexus** [comment]
Thanks drcynic. :+1: 

EDIT 25-04-16:
I've now forked a copy of the repo and have added rudimentary support for screensaver and mute command line parameters to the player module. In my working copy, I've also integrated support for storing context settings to the windows registry, and calling them. 

Thanks again for sharing the project @drcynic & @pixtur. I'm fairly new to working with C# myself, but this project has been an excellent learning experience for me, and the program itself has been a great learning tool also.

If there's anything that I can do to aid you guys with the T2 project, I'd be more than happy to help out.
